gitleaks.toml

```diff
@@ -0,0 +1,8 @@
+title = "MBI Scan"
+# Only using custom rules
```

Comment:
Thanks for switching to extend the default config! This comment can be dropped, and I think `useDefault` needs to be in the `[extend]` section as shown at https://github.com/gitleaks/gitleaks?tab=readme-ov-file#configuration

Reply:
Pushing correction now